docs/userGuide/siteConfiguration.md

```diff
@@ -33,7 +37,8 @@ Let's examine a typical `site.json` file:
 | Variable | Description |
 |----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **baseUrl** | The url base when you deploy the website. For example, if you are using Github Pages to host your deployed website, and the resulted page is `https://markbind.github.io/site-demo-cs2103`, then your `baseUrl` would be `/site-demo-cs2103` (relative to your domain, default is ` ` (empty)). Later you could use this variable to help you create top-level navigation link in the document. **You may need to change the `baseUrl` when deploy to different repo.** |
-| **pages** | An array of the pages config tells MarkBind all the source files that needs to be rendered. The `src` is the page to be rendered; `title` is the page title for the generated web page. |
+| **titlePrefix** | The prefix for all page titles. This prefix will be prepended to all rendered pages. |
+| **pages** | An array of the pages config tells MarkBind all the source files that needs to be rendered. The `src` is the page to be rendered; `title` is the page title for the generated web page. In addition, globs can be used to define pages that will be rendered. |

```

Comment:
Can you explain what happen if a page got collected by globs by don't have the title/frontmatter there?